ts/screens/messages/MessagesScreen.tsx

```diff
@@ -0,0 +1,178 @@
+  services: ServicesState;
+}>;
+
+export type OwnProps = Readonly<{
+  navigation: NavigationScreenProp<NavigationState>;
+}>;
+
+export type IMessageDetails = Readonly<{
+  item: Readonly<MessageWithContentPO>;
+  index: number;
+}>;
+
+export type Props = ReduxMappedProps & ReduxProps & OwnProps;
+
+const styles = StyleSheet.create({
+  tabBarUnderlineStyle: {
+    width: 0
+  },
+  activeTabStyle: {
+    borderBottomWidth: 2,
+    borderBottomColor: variables.brandPrimaryLight
+  },
+  NotActiveTabStyle: {
+    borderBottomWidth: 0,
+    borderBottomColor: variables.brandPrimaryInverted
+  }
+});
+
+/**
+ * This screen show the messages to the authenticated user.
+ *
+ * TODO: Just a mocked version at the moment.
+ * Going to be replaced with real content in @https://www.pivotaltracker.com/story/show/152843981
+ */
+class MessagesScreen extends React.Component<Props, never> {
+  private didFocusSubscription?: NavigationEventSubscription;
+
+  constructor(props: Props) {
+    super(props);
+  }
+
+  public componentDidMount() {
+    // TODO: Messages must be refreshed using pull-down @https://www.pivotaltracker.com/story/show/157917217
+    // tslint:disable-next-line no-object-mutation
+    this.didFocusSubscription = this.props.navigation.addListener(
+      "didFocus",
+      () => {
+        this.props.dispatch(loadMessages());
+      }
+    );
+  }
+
+  public componentWillUnmount() {
+    if (this.didFocusSubscription) {
+      this.didFocusSubscription.remove();
+      // tslint:disable-next-line no-object-mutation
+      this.didFocusSubscription = undefined;
+    }
+  }
+
+  private renderLoadingStatus = (isLoadingMessages: boolean) => {
+    return isLoadingMessages ? (
+      <ActivityIndicator size="small" color={variables.brandPrimary} />
+    ) : null;
+  };
+
+  public getServiceName = (senderServiceId: string): string => {
+    return this.props.services.byId[senderServiceId].service_name;
+  };
+
+  public getOrganizationName = (senderServiceId: string): string => {
+    return this.props.services.byId[senderServiceId].organization_name;
+  };
+  public getDepartmentName = (senderServiceId: string): string => {
+    return this.props.services.byId[senderServiceId].department_name;
+  };
+  public renderItem = (messageDetails: IMessageDetails) => {
+    return (
+      <MessageComponent
+        key={messageDetails.item.id}
+        date={messageDetails.item.created_at}
+        serviceOrganizationName={this.getOrganizationName(
+          messageDetails.item.sender_service_id
+        )}
+        serviceDepartmentName={this.getDepartmentName(
+          messageDetails.item.sender_service_id
+        )}
+        subject={messageDetails.item.subject}
+        navigation={this.props.navigation}
+        senderServiceId={messageDetails.item.sender_service_id}
+        markdown={messageDetails.item.markdown}
+        service={this.getServiceName(messageDetails.item.sender_service_id)}
```

Comment:
serviceName?

Reply:
Fixed whit next commit. Yes, this is a service name.